controllers/v1/entities.js

```diff
@@ -319,5 +319,60 @@ module.exports = class Entities extends Abstract {
     });
   }
 
+  /**
+  * @api {post} /kendra/api/v1/entities/getUsersByEntityAndRole.
+  * @apiVersion 1.0.0
+  * @apiName Get users by entityId and role.
+  * @apiGroup Entities
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
+  * @apiSampleRequest /kendra/api/v1/entities/getUsersByEntityAndRole
+  * @apiParamExample {json} Request-body:
+  * {
+    "entityId" : "5da829874c67d63cca1bd9d0",
+    "role" : "HM"
+  }
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
```

Comment:
@deepakr123 - Sample response is missing

Reply:
resolved